CodeEdit/Features/Editor/Views/EditorAreaView.swift

```diff
@@ -148,8 +148,6 @@ struct EditorAreaView: View {
                     editor.openTab(file: file)
                     editorManager.activeEditor = editor
                     focus = editor
-                    NSApp.activate(ignoringOtherApps: true)
-                    window?.makeKeyAndOrderFront(nil)
```

Comment:
This looks like a regression, was this meant to be removed?

Reply:
Yes, I meant to remove it in my other PR but forgot to. We don't want CodeEdit to come into focus when dragging external files to it in case the user wants to drag multiple files in. This is standard when compared to other apps. I did not notice this initially.